osbuild/util/ostree.py

```diff
@@ -215,15 +214,24 @@ def parse_input_commits(commits):
 def deployment_path(root: PathLike, osname: str, ref: str, serial: int):
     """Return the path to a deployment given the parameters"""
 
-    base = os.path.join(root, "ostree")
+    if osname == "" and ref == "" and serial == None:
```

Comment:
Is there even a use case in osbuild for having multiple deployments in the image? I'd just nuke those parameters and always do the glob approach. That should simplify some of the callers currently doing extra work to prepare those params.

Obviously fine to do as a follow-up.

Reply:
yeah, I think there is an argument to make for that - +1 for a follow up. We'd need to keep supporting specifying the options even though they would do nothing.